### DISCUSSION TITLE
动画中出现圆环、流动的光有很多锯齿

### DISCUSSION THREAD

**Discussion by coderyqy** [Tencent/libpag#2399] [Q&A]
问题展示：
![image](https://github.com/user-attachments/assets/d68a7880-c3d2-42c6-b303-f8910b31c21d)

请问在程序上有优化的可能吗？如何排查问题？

**Comment by kevingpqi123** [comment]
这类问题建议将 AE 工程发出来，我们看下具体是啥原因

**Comment by jinwuwu001** [comment]
用户长期无进一步反馈，暂时关闭，有需要可以再开启
